persistent/ChangeLog.md

```diff
@@ -1,4 +1,9 @@
-# Changelog for persistent
+# Changelog for persistentChan
+
+## 2.14.6.1 (unreleased)
+
+* [#1518](https://github.com/yesodweb/persistent/pull/1518)
+    * Normalize postgres type aliases to prevent noop migrations
```

Comment:
The change is actually in `persistent-postgresql`, not `persistent`, so you'll want to make the version bump and changelog modification there 😄

Reply:
thanks for fixing it :)